server/src/services/auth.service.ts

```diff
@@ -269,19 +276,16 @@ export class AuthService {
         (ou) => ou.organizationId === user.defaultOrganizationId
       );
 
-      if (!organizationUser) {
+      if ((user?.organizationUsers, !organizationUser)) {
```

Comment:
What is the use of this operation here?

Reply:
that was a mistake, will remove that part